pkg/relayer/relayminer_test.go

```diff
@@ -57,3 +60,69 @@ func TestRelayMiner_StartAndStop(t *testing.T) {
 	err = relayminer.Stop(ctx)
 	require.NoError(t, err)
 }
+
+func TestRelayMiner_Ping(t *testing.T) {
```

Comment:
Would you mind adding one more test for the error cases where the relayer proxy mock's `#PingAll()` returns the temporary and non-temporary `*url.Error`s such that we can assert (and cover regression of) the resulting error returned from the HTTP GET on the ping endpoint?

Reply:
Agree. Added here https://github.com/eddyzags/poktroll/commit/a100b3f8efecdd93d1a68dd11da57605a1653cc0